design_introspection-plugin/.clang-format

```diff
@@ -0,0 +1,151 @@
+---
```

Comment:
Why is this being added in this subdirectory?

Reply:
I will finally work on the clang format PR to have it apply to all plugins.
I will remove it from this PR